jobs/aws/eks-anywhere-build-tooling/hook-presubmit.yaml

```diff
@@ -54,10 +54,10 @@ presubmits:
           periodSeconds: 10
         resources:
           requests:
-            memory: "16Gi" # Higher memory for kernel builds
+            memory: "26Gi" # Higher memory for kernel builds
```

Comment:
Double check that this a valid instance configuration for fargate. Are these builds happening in buildkitd or on the main container?  You may want to add a resource sectional to the buildkitd container to make it easier to validate when added together it falls within a valid fargate range

https://docs.aws.amazon.com/AmazonECS/latest/developerguide/AWS_Fargate.html#fargate-tasks-size

Reply:
So I checked we had 26Gi for some of our EKS-D jobs, so i think its a valid?
Also the build is run using `buildkit`, so sounds we might need something over there.

Reply:
Yeah 4cpu and 26 mem should be ok, but I'm honestly not sure what happens when buildkit has no requests? I would drop 512 or so from the CPU and give it to buildkitd at least, and give it 1gb of men. If the bulk of the work for this build is actually happening in the buildkitd, like from a dockerfile, then you may actually want most of the resources on that container instead?